docs/infra/set-up-monitoring-notifications.md

```diff
@@ -0,0 +1,65 @@
+# Set up monitoring notifications
+
+## Overview 
+
+The Monitoring Module is responsible for presenting monitoring information from our application and infrastructure. It helps us identify performance bottlenecks, potential issues, and overall system health. With this module, we gain valuable insights to make data-driven decisions and ensure the reliability and availability of our services.
+
+## Features
+
+* Real time alerts: Module supports alerting rules to notify stakeholders when specific thresholds are breached.
+* Notification channels: it supports various notification channels, specifically email alerts notifications and integration with external Incident management tools like Splunk-On-Call or Pagerduty.
```

Comment:
```suggestion
The monitoring module defines metric-based alerting policies that provides awareness into issues with the cloud application. The module supports integration with external incident management tools like Splunk-On-Call or Pagerduty. It also supports email alerts.
```

the overview is a bit wordy, provided a suggestion to make it concise. let's combine the features section with the overview section too. also, we don't need to capitalize monitoring module.